src/js/rest/putCustomize.js

```diff
@@ -0,0 +1,54 @@
+import createError from './common/createError';
```

Comment:
Because customize is a verb, so the function name should be changed
putCustomize => updateCustomization